### PR TITLE
Converted the serverless to terraform and refactored the both serverless and terraform code

### PR DESCRIPTION
**Pull request by ewuharun**
Fixes #145 

- [ ] Does not depend on other PRs
- [x] Depends on other PR being merged first: #131 

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [x] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
I have implemented the terraform task
## What was done
1. s3->sqs
2. sqs->video compression->dynamo
3. dynamo->SNS->Discord
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: src/main/java/com/serverless/lambdas/TerraformLambda.java [hunk 1]
```diff
@@ -0,0 +1,253 @@
+package com.serverless.lambdas;
```
[the remark this answers is not in the document]
Comment (mhazad) [reply src/main/java/com/serverless/lambdas/TerraformLambda.java:1]: You are violating DRY. Copying code can be considered the worst possible mistake by a developer. Please delete copied code and reuse the existing one.

**Review by mhazad — CHANGES_REQUESTED** [review]
Please address the feedback.

File: src/main/java/com/serverless/lambdas/TerraformLambda.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ewuharun) [reply src/main/java/com/serverless/lambdas/TerraformLambda.java]: Deleted the TerraformLambda.java

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**